vhdbuilder/packer/install-dependencies.sh

```diff
@@ -257,64 +257,38 @@ for NGINX_VERSION in ${NGINX_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
-# pull patched hyperkube image for AKS
-# this is used by kube-proxy and need to cover previously supported version for VMAS scale up scenario
-# So keeping as many versions as we can - those unsupported version can be removed when we don't have enough space
-# below are the required to support versions
-# v1.18.14
-# v1.18.17
-# v1.19.7
-# v1.19.9
-# v1.20.2
-# v1.20.5
-# NOTE that we keep multiple files per k8s patch version as kubeproxy version is decided by CCP.
-PATCHED_HYPERKUBE_IMAGES="
+KUBE_PROXY_IMAGE_VERSIONS="
 1.17.13
+1.17.13-hotfix.20210310.1
 1.17.16
+1.17.16-hotfix.20210310.1
 1.18.8-hotfix.20200924
+1.18.8-hotfix.20201112.1
 1.18.10-hotfix.20210118
+1.18.10-hotfix.20210310.1
 1.18.14-hotfix.20210118
 1.18.14-hotfix.20210322
 1.18.17-hotfix.20210322
 1.19.1-hotfix.20200923
+1.19.1-hotfix.20200923.1
 1.19.3
 1.19.6-hotfix.20210118
+1.19.6-hotfix.20210310.1
 1.19.7-hotfix.20210310
+1.19.7-hotfix.20210310.1
 1.19.9-hotfix.20210322
 1.20.2
 1.20.2-hotfix.20210310
+1.20.2-hotfix.20210310.1
```

Comment:
none of these new stuff are needed

Reply:
In case we need these, can you also please run a vhd build for gen2 & see if that succeeds, as I'm afraid we may exceed the 30gb limit with so many new versions added